pkg/commands/cmdlib/bundleio.go

```diff
@@ -202,3 +204,40 @@ func formatFromFile(path string) string {
 	}
 	return ""
 }
+
+// ParseAnnotations parses CLI flag format for annotations
+func ParseAnnotations(p string) map[string]string {
+	if p == "" {
+		return nil
+	}
+
+	m := make(map[string]string)
+	splat := strings.Split(p, ";")
+	for _, v := range splat {
+		kv := strings.Split(v, ",")
+		if len(kv) == 2 {
+			m[kv[0]] = kv[1]
+		}
+	}
+	return m
+}
+
+// MergeOptions reads multiple options files and combines them into a single map.
+// Option values will be overwritten if a later file has the same key.
```

Comment:
I think it's probably clearer to produce an error if there's a duplicate key. imo, it's confusing behavior to overwrite.

Reply:
Well, I am thinking for this for specialization. A global options file, then individual options files that may override the global ones.